### PR TITLE
merge request, feat(inapp_controls) --> develop SDK-1033

### PR DESCRIPTION
**Pull request by piyush-kukadiya**
- Adds public methods for suspending/discarding & resuming InApp Notifications

### CODE REVIEW

**Review by darshanclevertap — APPROVED** [review]
LGTM 👍🏾

File: index.js [hunk 1]
```diff
@@ -697,6 +697,38 @@ var CleverTap = {
...
+     * Resumes display of InApp Notifications.
+     *
+     * If suspendInAppNotifications() was called previously, calling this method will instantly show
+     * all queued InApp Notifications and also resume InApp Notifications on events raised after this
+     * method is called.
+     *
+     * If discardInAppNotifications() was called previously, calling this method will only resume
+     * InApp Notifications on events raised after this method is called.
+     */
+    resumeInAppNotifications: function () {
```
[the remark this answers is not in the document]
Comment (piyush-kukadiya) [reply index.js:728]: Done

**Review by Aditi3 — APPROVED** [review]
👍🏼

**Review by akashvercetti — APPROVED** [review]

**Comment by Aditi3** [comment]
@piyush-kukadiya Could you resolve the conflicts?

**Pull request merged**